pkg/identity/numericidentity.go

```diff
@@ -71,6 +75,41 @@ var (
 	}
 )
 
+// IsUserReservedIdentity returns true if the given NumericIdentity belongs
+// to the space reserved for users.
+func IsUserReservedIdentity(id NumericIdentity) bool {
+	return id.Uint32() >= UserReservedNumericIdentity.Uint32() &&
+		id.Uint32() < MinimalNumericIdentity.Uint32()
+}
+
+// AddReservedNumericIdentity adds the given numeric identity and respective
+// label to the list of reservedIdentities. If the numeric identity is not
+// between 128 and 256 it will not be added.
+func AddReservedNumericIdentity(identity NumericIdentity, label string) {
+	if !IsUserReservedIdentity(identity) {
+		return
```

Comment:
Please return an error instead and leave it up to the caller whether to ignore the error or not